tools/c7n_azure/c7n_azure/utils.py

```diff
@@ -473,8 +472,14 @@ def __str__(self):
             return self.str_value
 
     @staticmethod
-    def duration_from_period_and_units(period, retention_period_unit):
+    def iso8601_duration(period, retention_period_unit):
         iso8601_str = "P{}{}".format(period, retention_period_unit.iso8601_symbol)
+        return iso8601_str
+
+    @staticmethod
+    def duration(period, retention_period_unit):
+        iso8601_str = RetentionPeriod.iso8601_duration_from_period_and_units(period,
```

Comment:
`iso8601_duration`? (And if yes, we need tests for these functions :))

Reply:
Yup! the `duration` function isn't actually used anywhere anymore. I'll delete it. And I'll also add some tests for the rest of this helper class.